tools/zbdump

```diff
@@ -46,8 +46,12 @@ parser.add_argument('-n', '--count', action='store', type=int, default=-1)
 parser.add_argument('-D', action='store_true', dest='showdev')
 args = parser.parse_args()
 
-if args.showdev:
-    show_dev()
+if args.showdev and not args.devstring:
+    print >>sys.stderr, "ERROR: Must specify an interface."
```

Comment:
@whoot previously it was permitted for it to show all available interfaces in this case -- it shouldn't be just allowing you to do a -D if you provide a device string -- at least I always interpreted this behavior to be similar to zbid (and honestly am not sure if we really need showdev in most of these programs, as zbid should handle that for people).

Reply:
Oops, that is indeed a copy&paste mistake. I will correct that immediately.

Reply:
The -D without -i is again due to the usage of a Sewio.

Reply:
exactly. But why shouldn´t I fix problems that are Sewio specific in the Sewio branch?